src/main/java/com/google/api/codegen/transformer/java/JavaGapicSurfaceTransformer.java

```diff
@@ -122,23 +128,26 @@ public JavaGapicSurfaceTransformer(
               JavaFeatureConfig.newBuilder()
                   .enableStringFormatFunctions(enableStringFormatFunctions)
                   .build());
-      StaticLangApiFileView apiFile = generateApiFile(context);
+      StaticLangFileView<StaticLangApiView> apiFile = generateApiFile(context);
```

Comment:
So the thing that drove me to do this was that there were two classes, `StaticLangApiFileView` and `StaticLangSettingsFileView`, which were identical except for the type of the view for the class they contained, and I needed to add two more types of files, 1) stub interface and 2) grpc stub implementation. I think it would harm code navigation a lot to just put a generic type like `Object` or `ViewModel` as the type of `classView()`, even if I didn't need to call methods on `classView`. That's the reason everything else is explicit types too; the Snippets language doesn't care, and everything could be an `Object` and the Snippets rendering would work fine.